[2023-10-17T11:19:37.457+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-17T11:19:37.466+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-17T11:19:37.466+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:37.478+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-15 00:00:00+00:00
[2023-10-17T11:19:37.486+0000] {standard_task_runner.py:57} INFO - Started process 704 to run task
[2023-10-17T11:19:37.488+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-15T00:00:00+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpel8jwprp']
[2023-10-17T11:19:37.489+0000] {standard_task_runner.py:85} INFO - Job 139: Subtask waiting_for_data
[2023-10-17T11:19:37.532+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:37.611+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-15T00:00:00+00:00'
[2023-10-17T11:19:37.624+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:37.624+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:37.624+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:37.624+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:37.634+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210515T000000, start_date=20231017T111937, end_date=20231017T111937
[2023-10-17T11:19:37.662+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:37.681+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:43:14.763+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-18T10:43:14.770+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-18T10:43:14.771+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:43:14.820+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-15 00:00:00+00:00
[2023-10-18T10:43:14.827+0000] {standard_task_runner.py:57} INFO - Started process 1675 to run task
[2023-10-18T10:43:14.830+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-15T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpgn53y_ux']
[2023-10-18T10:43:14.831+0000] {standard_task_runner.py:85} INFO - Job 138: Subtask waiting_for_data
[2023-10-18T10:43:14.882+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:43:14.960+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-15T00:00:00+00:00'
[2023-10-18T10:43:14.976+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:43:14.980+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210515T000000, start_date=20231018T104314, end_date=20231018T104314
[2023-10-18T10:43:14.997+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 138 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1675)
[2023-10-18T10:43:15.043+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:43:15.077+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:43:31.706+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-18T11:43:31.722+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-18T11:43:31.723+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:43:31.747+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-15 00:00:00+00:00
[2023-10-18T11:43:31.758+0000] {standard_task_runner.py:57} INFO - Started process 996 to run task
[2023-10-18T11:43:31.763+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-15T00:00:00+00:00', '--job-id', '230', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp5nij82j1']
[2023-10-18T11:43:31.764+0000] {standard_task_runner.py:85} INFO - Job 230: Subtask waiting_for_data
[2023-10-18T11:43:31.830+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:43:31.964+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-15T00:00:00+00:00'
[2023-10-18T11:43:31.990+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:43:31.991+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:43:31.994+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:43:31.994+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:43:32.033+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210515T000000, start_date=20231018T114331, end_date=20231018T114332
[2023-10-18T11:43:32.099+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:43:32.196+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:44:27.139+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-19T12:44:27.167+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [queued]>
[2023-10-19T12:44:27.167+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:44:27.203+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-05-15 00:00:00+00:00
[2023-10-19T12:44:27.212+0000] {standard_task_runner.py:57} INFO - Started process 1205 to run task
[2023-10-19T12:44:27.241+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-05-15T00:00:00+00:00', '--job-id', '264', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpt0vtm5rj']
[2023-10-19T12:44:27.243+0000] {standard_task_runner.py:85} INFO - Job 264: Subtask waiting_for_data
[2023-10-19T12:44:27.330+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-05-15T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:44:27.469+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-15T00:00:00+00:00'
[2023-10-19T12:44:27.491+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:44:27.491+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:44:27.491+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:44:27.491+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:44:27.509+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210515T000000, start_date=20231019T124427, end_date=20231019T124427
[2023-10-19T12:44:27.562+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:44:27.663+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
